internal/pkg/cli/svc.go

```diff
@@ -18,8 +18,8 @@ func BuildSvcCmd() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "svc",
 		Short: "Service commands.",
-		Long: `Command for working with services.
-A service represents an Amazon ECS service or task.`,
+		Long: `Command for working with long-running services.
+		A service represents an Amazon ECS service.`,
```

Comment:
we should remove the extra space here, otherwise it looks awkward in the help menu i think